src/Symfony/Component/Security/Core/Resources/translations/security.fa.xlf

```diff
@@ -4,43 +4,43 @@
         <body>
             <trans-unit id="1">
                 <source>An authentication exception occurred.</source>
-                <target>خطایی هنگام تعیین اعتبار اتفاق افتاد.</target>
+                <target>خطایی هنگام احراز هویت رخ داده است.</target>
             </trans-unit>
             <trans-unit id="2">
                 <source>Authentication credentials could not be found.</source>
-                <target>شرایط تعیین اعتبار پیدا نشد.</target>
+                <target>شرایط احراز هویت یافت نشد.</target>
```

Comment:
Google tells me that this translates as *"authentication conditions could not be found"*. Indeed, this phrase contains "شرایط" which Google translates as "conditions" instead of "credentials".

Reply:
Hello @javiereguiluz .
Thank you for reviewing this request.

Reply:
The two words, terms, and credentials do not have the same meaning.
Google's translation service is very good, but there are still some issues that I hope to be resolved in the future.

Reply:
The two words, conditions and credentials do not have the same meaning.
Google's translation service is very good, but there are still some issues that I hope to be resolved in the future.

Reply:
do the ending point be at the other side? as it is a rl language